### DISCUSSION TITLE
Split Phase sawtooth reading?

### DISCUSSION THREAD

**Discussion by pengsloth** [emporia-vue-local/esphome#249] [Q&A]
I'm having some very strange reading from a dual 20A breaker in my box. It goes to the "guest house" attached to the main house and I've determined that it powers the kitchen (yes the entire thing) in there. I tried putting a clamp on just one, and then the other, and then finally seeing from another discussion to read from a unbalanced load clamping both with one running through backwards. In each case I still get this strange sawtooth like reading.

<img width="1076" alt="Screenshot 2024-01-03 205032" src="https://github.com/emporia-vue-local/esphome/assets/22439182/ae7b63a3-e020-45fb-8ae4-c3d010fc266e">

Any ideas?

**Comment by flaviut** [comment]
Not sure what the answer is here (maybe someone who has experienced this can chime in?) but a good first step for debugging is deleting the filters for this circuit (`filters: [...]`) from the config and re-uploading.
